bin/grocery-delivery

```diff
@@ -120,7 +127,12 @@ def partial_upload(knife, repo, checkpoint, local_head)
   end
 
   knife.cookbook_delete(deleted_cookbooks) if deleted_cookbooks
```

Comment:
`berks` doesn't have a notion of deleting cookbooks, so don't think we can do anything with that. Perhaps they'll add that feature down the line.